glide.lock

```diff
@@ -1191,7 +1191,7 @@ imports:
 - name: github.com/spf13/cast
   version: e31f36ffc91a2ba9ddb72a4b6a607ff9b3d3cb63
 - name: github.com/spf13/cobra
-  version: c439c4fa093711d42e1b01acb1235b52004753c1
```

Comment:
Don't bump this, we need to use the same version as k8s is. That will be the reason for your unnecessary completion update.